### PR TITLE
cave : Cleanup duplicates, Notes

### PR DESCRIPTION
**Pull request by cam900**

### CODE REVIEW

File: src/mame/video/cave.cpp [hunk 1]
```diff
@@ -1632,6 +1565,7 @@ uint32_t cave_state::screen_update_cave(screen_device &screen, bitmap_rgb32 &bit
 
 void cave_state::cave_get_sprite_info(int chip)
 {
+	int spriteram_bankmax = m_spriteram.bytes() / 0x4000;
```
Comment (Osso13) [inline src/mame/video/cave.cpp]: Compiling src/mame/video/cave.cpp...
../../../../../src/mame/video/cave.cpp: In member function 'void cave_state::cav
e_get_sprite_info(int)':
../../../../../src/mame/video/cave.cpp:1568:38: error: 'optional_shared_ptr_arra
y<short unsigned int, 4> {aka class object_array_finder<shared_ptr_finder<short
unsigned int, false>, 4>}' has no member named 'bytes'
  int spriteram_bankmax = m_spriteram.bytes() / 0x4000;

File: src/mame/drivers/cave.cpp [hunk 2]
```diff
@@ -5334,14 +5268,13 @@ DRIVER_INIT_MEMBER(cave_state,ppsatan)
...
 {
-	uint8_t *ROM = memregion("audiocpu")->base();
 	uint8_t *src = memregion("sprites0")->base();
 	int len = memregion("sprites0")->bytes();
 	int i, j;
 
 	init_cave();
 
-	membank("z80bank")->configure_entries(0, 8, &ROM[0x00000], 0x4000);
+	init_z80_bank()
```
Comment (Osso13) [inline src/mame/drivers/cave.cpp]: you forgot a semicolon

**Comment by Tafoid** [comment]
ddonpach, donpachi and clones - cave.cpp
Graphic corruption

ddonpach-diff
donpachi-diff

-----

agallet, sailormn, mazinger and clones
Failure to boot to title

**Comment by galibert** [comment]
Fixed in db658231bba8ae6221d24581a5638e9318c1490f and 4ee5b635a5d798e7743e85a92154936b5cf62f0f

The LOAD16 is not endian-safe though.  Whatever.

**Pull request merged**